packages/wallets/src/features/cfd/__tests__/CFDPlatformsList.spec.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { useActiveWalletAccount } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import useDevice from '../../../hooks/useDevice';
+import CFDPlatformsList from '../CFDPlatformsList';
+
+jest.mock('@deriv/api-v2', () => ({
+    ...jest.requireActual('@deriv/api-v2'),
+    useActiveWalletAccount: jest.fn(() => ({
+        data: {
+            currency_config: {
+                is_crypto: false,
+            },
+        },
+    })),
+}));
+
+jest.mock('../../../hooks/useDevice', () => jest.fn(() => ({ isMobile: false })));
+
+const mockPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: jest.fn(() => ({ push: mockPush })),
+}));
+
+// eslint-disable-next-line react/display-name
+jest.mock('../CFDPlatformsListEmptyState', () => () => <div>CFDPlatformsListEmptyState</div>);
```

Comment:
Nit @george-usynin-deriv 
```suggestion
jest.mock('../CFDPlatformsListEmptyState', () => jest.fn(() => <div>CFDPlatformsListEmptyState</div>));
```